tests/test_websocket_notifications.py

```diff
@@ -58,7 +58,9 @@ def test_registration_update_notification(edge, cloud_api, websocket):
 
 
 def test_registration_notification(edge, cloud_api, websocket):
-    wait_time = 3 * 60
+    # Wait time needs to be quite long, with the LmP OSTree image verifications - boot time is long
```

Comment:
Update the Copyright statement at the top of this file.

Reply:
```
# ----------------------------------------------------------------------------
# Copyright (c) 2022 Izuma Networks
# Copyright (c) 2020-2021, Pelion and affiliates.
#
# SPDX-License-Identifier: Apache-2.0
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
# ----------------------------------------------------------------------------

```
That's what in the header right now, is that good or what change were you thinking of @petedyerarm ?

Reply:
Aligned the copyrights now. Newest on top, comma betweeen year and company name.